tests/fv3_conf/cpld_control_run.IN

```diff
@@ -96,6 +96,10 @@ else
        RFILE_OLD=$(basename $RFILE)
           mv -f $RFILE INPUT/"${RFILE_OLD//${RESTART_FILE_PREFIX}./}"
      done
+     echo "IAU_OFFSET=$IAU_OFFSET, FHROT=$FHROT"
+     if [[ $IAU_OFFSET == 6 ]] && [[ $FHROT -le 6 ]]; then
+       cp /scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/inc/c96/atm/fv_increment*.nc ./INPUT/.
```

Comment:
@jkbk2004 I need UFS code managers to copy these data to input-data-20221101/FV3_input_data/INPUT_L127_mx100/.